Tests/FeaturevisorSDKTests/InstanceTests.swift

```diff
@@ -384,4 +384,172 @@ class FeaturevisorInstanceTests: XCTestCase {
         // enabling required should enable the feature too
         XCTAssertTrue(sdk.isEnabled(featureKey: "myKey"))
     }
+
+    // should be disabled because required has different variation
+    func testShouldHonourRequiredFeaturesWithVariationDisabled() {
+
+        // GIVEN
+        var options: InstanceOptions = .default
+        options.datafile = DatafileContent(
```

Comment:
is there any Swift-ish way of having static methods from structs, like `DatafileContent.fromJSON("{ ...datafileJSONAsStringHere }")`?

just thinking out loud, in case that helped copying the already available JSON content in JS SDK's tests.

Reply:
We can load a JSON directly from a file. We are using this approach for types unit tests. If we would like to have JSON as a string it would need to be an escaped string, so something like this:
` let jsonString = "{\"schemaVersion\":\"1\",\"revision\":\"0.0.666\",\"attributes\":[],\"segments\":[],\"features\":[]}"`